src/i18n.ts

```diff
@@ -1,55 +1,72 @@
 import LanguageDetector from "i18next-browser-languagedetector";
 
 export const languages = [
-  { lng: "en", label: "English", data: require("./locales/en.json") },
-  { lng: "bg-BG", label: "Български", data: require("./locales/bg-BG.json") },
-  { lng: "de-DE", label: "Deutsch", data: require("./locales/de-DE.json") },
-  { lng: "es-ES", label: "Español", data: require("./locales/es-ES.json") },
-  { lng: "ca-ES", label: "Catalan", data: require("./locales/ca-ES.json") },
-  { lng: "el-GR", label: "Ελληνικά", data: require("./locales/el-GR.json") },
-  { lng: "fr-FR", label: "Français", data: require("./locales/fr-FR.json") },
+  { lng: "en", label: "English", data: "en.json" },
+  { lng: "bg-BG", label: "Български", data: "bg-BG.json" },
+  { lng: "de-DE", label: "Deutsch", data: "de-DE.json" },
+  { lng: "es-ES", label: "Español", data: "es-ES.json" },
+  { lng: "ca-ES", label: "Catalan", data: "ca-ES.json" },
+  { lng: "el-GR", label: "Ελληνικά", data: "el-GR.json" },
+  { lng: "fr-FR", label: "Français", data: "fr-FR.json" },
   {
     lng: "id-ID",
     label: "Bahasa Indonesia",
-    data: require("./locales/id-ID.json"),
+    data: "id-ID.json",
   },
```

Comment:
This should reformat back to one line if you delete the newline after the `{`.

```js
  { lng: "id-ID", label: "Bahasa Indonesia", data: "id-ID.json" },
```

Reply:
I would expect prettier to handle that. How come it didn't?